src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -1,5 +1,3 @@
-<dot-edit-layout-advanced [pageState]="pageState | async" *ngIf="(pageState | async)?.template && isAdvancedTemplate | async else designer"></dot-edit-layout-advanced>
+<!--Advance template support was removed on commit https://github.com/dotCMS/core-web/pull/589-->
+<dot-edit-layout-designer [pageState]="pageState | async"></dot-edit-layout-designer>
 
-<ng-template #designer>
-    <dot-edit-layout-designer [pageState]="pageState | async"></dot-edit-layout-designer>
-</ng-template>
```

Comment:
I thought we would not remove any code

Reply:
In this case we have to, I don't think is a good idea to add an attr to this nav bar to said if we're going to edit layout or template because I don't think this nav bar should care about that.